.prettierignore

```diff
@@ -1,3 +1,4 @@
 dist
 node_modules
-codegen
\ No newline at end of file
+codegen
+src/pinecone-generated-ts-fetch
```

Comment:
I think this was lost when I was trying to reconcile the different feature branches.